pydantic/types.py

```diff
@@ -238,7 +238,7 @@ def validate(cls, value: Any) -> Any:
 
 
 class ConstrainedNumberMeta(type):
-    def __new__(cls, name: str, bases: Any, dct: Dict[str, Any]) -> 'ConstrainedInt':
+    def __new__(cls, name: str, bases: Any, dct: Dict[str, Any]) -> 'ConstrainedInt':  # type: ignore
```

Comment:
Mypy doesn't like the return type any more; alternatively we could just not cast the result to be a `'ConstrainedInt'`; I'm not sure which is preferable.

Also, mypy now supports more detailed `type: ignore` messages (see the section on error codes [here](http://mypy-lang.blogspot.com/)). Might be worth making modifications to use those where appropriate.